Recognize JPG type to display in download list

tkohr opened the pull request:
PR recognizes JPG file type in links with protocol `WWW:DOWNLOAD` and displays it in the download list, which is currently not the case ("unknown"). To test this behavior, search for entry "Schémas de cohérence territoriale dans l'Oise" (471011ac-b927-4c8d-9589-ce3c530ea08d) for example.

fgravin commented:
Thanks @tkohr, please add a description, with an exemple of a failing link (you can take the one i wrote in Jira).
Also, you must add a test which would break without your fix and which now passes.

Thanks

tkohr commented:
Yes right @fgravin, I added the missing tests for the file types.

fgravin commented on libs/feature/record/src/lib/links/link-utils.spec.ts:
```diff
@@ -4,24 +4,86 @@ import {
   describe('#getDownloadFormat', () => {
-    describe('for a FILE link', () => {
+    describe('for a csv FILE link', () => {
       it('returns download format', () => {
```
```suggestion
      it('returns csv format', () => {
```

fgravin commented on libs/feature/record/src/lib/links/link.fixtures.ts:
```diff
@@ -0,0 +1,108 @@
+export const LINK_FIXTURES = {
```
good !

fgravin reviewed (changes requested):
minor comment but it looks good to me thanks.
You'll have to wait for the big refactoring PR to be merged and then update your PR accordingly, sorry for that.

tkohr commented:
You are right @fgravin, the description wasn't quite clear. Updated that, rebased and integrated your suggestion. If CI ends up green and you are ok with it, ready to merge I think.
